diff --git a/webrtc.gni b/webrtc.gni
--- a/webrtc.gni
+++ b/webrtc.gni
@@ -336,9 +336,11 @@ rtc_libvpx_dir = "//third_party/libvpx"
 rtc_opus_dir = "//third_party/opus"

-# Desktop capturer is supported only on Windows, OSX and Linux.
-rtc_desktop_capture_supported =
-    (is_win && current_os != "winuwp") || is_mac ||
-    ((is_linux || is_chromeos) && (rtc_use_x11_extensions || rtc_use_pipewire))
+declare_args() {
+  # Desktop capturer is supported only on Windows, OSX and Linux.
+  rtc_desktop_capture_supported =
+      (is_win && current_os != "winuwp") || is_mac ||
+      ((is_linux || is_chromeos) && (rtc_use_x11_extensions || rtc_use_pipewire))
+}

 ###############################################################################
 # Templates
